quickstart.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
```

Comment:
Did you try with an official python image as a base image? Maybe make sense if it works, since could simplify image maintenance.

Reply:
I think I should, actually for elasticsearch 8.5 its giving error (unrecognized service) when we try to start the elasticsearch so I'm thinking to change the image to Python 3.9 or may be I'll use ES 8.5 image and test the same.